kratos/includes/node.h

```diff
@@ -778,6 +787,27 @@ class Node : public Point,  public IndexedObject, public Flags
         KRATOS_ERROR <<  buffer.str() << std::endl;
     }
 
+    /**
+     * @brief Faster get with very simple and plain check in debug, in release it is direct
+     * @param Position Position of the DoF
+     */
+    inline const DofType& FastGetDof(const IndexType Position) const

```

Comment:
this method is very risky in the case you have dofs coming from different modelparts with different dof lists. 
We deliberately avoided to add this, as you have no way (not even in debug) to check that the variable you want is the correct one.

I will block for now ... but let's see if @pooyan-dadvand has a different point of view on this

Reply:
@RiccardoRossi for example a scheme that uses only one ModelPart as input this is right. Let the developer decide. Here you are increasing computational cost for everyone, for the sake of generality; but then complaining when I add several "ifs" for the sake of generality

Reply:
From my point of view, i would define FastGetDof(Variable, Position)

and then just to nothing with the variable. At least you could use it in debug mode to see that everythign is fine. Such solution would have no overhead